lib/services/common-api-presenter.js

```diff
@@ -300,7 +300,7 @@ function CommonApiPresenterFactory(
                             // If we failed to render the error then we've got larger problems.
 
                             logger.error('Unable to render error template.', {
-                                macaddress: macaddress,
+                                macaddress: options.macaddress,
```

Comment:
macaddress is `undefined` in current scope and even closure, so this deserve a fix.